common/overlaysynccleanuphandler.h

```diff
@@ -43,6 +43,15 @@ class OverlaySyncCleanupHandler : public CleanUpHandler
      * @return ReturnCode::FAILURE on failure
      */
     ReturnCode clean() override;
+
+    /**
+     * @brief this function is needed to query member name yet it is not needed for
+     * OverlaySyncCleanupHandler. Since OverlaySyncCleanupHandler uses DirectoryCleanUpHandler
+     * for removing directory, this function won't be called ever.
```

Comment:
1. Only use one short sentence for @brief, write the longer description in a separate paragraph.
2. This function **is** called when we iterate over the cleanuphandlers, but the reasoning is good anyway.